code/game/objects/items.dm

```diff
@@ -117,6 +117,12 @@ GLOBAL_DATUM_INIT(fire_overlay, /image, image("icon" = 'icons/goonstation/effect
 	var/in_inventory = FALSE //is this item equipped into an inventory slot or hand of a mob?
 	var/tip_timer = 0
 
+	// item hover FX
+	/// Is this item inside a storage object?
+	var/in_storage = FALSE
```

Comment:
I just noticed we have a variable `in_inventory` which is *very* similar to this var. That one gets set to true on equip, pickup, and when placed in storage. Basically when the item is in a place where you'd want the outline. Might be better to use that one.

Reply:
We actually check for both here, because `in_inventory` does not cover items that are on the ground or items that are nested within storage containers.

Reply:
`in_inventory` should handle *all* cases where an item is inside storage. If you look at the bottom of 
`/obj/item/storage/proc/handle_item_insertion`, it gets set to true there. I can't imagine why it wouldn't work if the storage item is on the ground, or with nesting. Unless there's something I'm not seeing.

Reply:
Hmm, there might be something funky happening somewhere then, because it wasn't working properly with just that when I was testing.